android/app/build.gradle

```diff
@@ -51,7 +52,7 @@ android {
 
     defaultConfig {
         applicationId "com.flutterplaza.finflow"
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
Yes it can't, as per the [documentation](https://facebook.meedu.app/docs/5.x.x/android) latest version of the flutter_facebook_auth package requires an Android SDK that's >=21